modules/real_time_enforcer/variables.tf

```diff
@@ -70,20 +70,24 @@ variable "network_project" {
 
 variable "enforcer_ssh_allow_ranges" {
   description = "List of CIDRs that will be allowed ssh access to forseti real time enforcer"
-  type        = "list"
+  type        = list(string)
   default     = ["0.0.0.0/0"]
 }
 
 variable "enforcer_instance_metadata" {
   description = "Metadata key/value pairs to make available from within the real time enforcer instance."
-  type        = "map"
+  type        = map(string)
   default     = {}
 }
 
 variable "enforcer_instance_access_config" {
   description = "Enforcer instance 'access_config' block"
-  type        = "map"
-  default     = {}
+  default     = null
+  type = object({
```

Comment:
Should this also be `map(any)` to correspond with the similar change to the other access configuration variables?